src/Param.tsx

```diff
@@ -51,7 +52,7 @@ export function GraphParam<S extends Schema, K extends keyof S>(
 			fontSize={fontSize}
 			dominantBaseline="middle"
 			>
-			{props.param}
+			{kinds[node.kind].params[param].label}
```

Comment:
Nit: Same as in Input.tsx. Maybe this could be done in the parent component? But not really necessary at the moment. Let me know what you think.